### PR TITLE
Implement Simplified Chinese localization for the Casdon TD Pro 2.

### PR DESCRIPTION
**Pull request by jianyun8023**
Implement Simplified Chinese localization for the Casdon TD Pro 2.
Added English configuration, other multilingual configurations not added.

### CODE REVIEW

File: custom_components/tuya_local/devices/casdon_td_pro_2.yaml [hunk 1]
```diff
@@ -158,47 +160,47 @@ secondary_entities:
...
+            value: cooking
           - dps_val: cancel
-            value: Cooking Canceled
+            value: cancel
           - dps_val: done
-            value: Cooking Completed
+            value: done
           - dps_val: pause
-            value: Cooking Paused
+            value: pause
```
Comment (jianyun8023) [inline custom_components/tuya_local/devices/casdon_td_pro_2.yaml:203]: The retention of value here is necessary for enum usage.

File: custom_components/tuya_local/devices/casdon_td_pro_2.yaml [hunk 2]
```diff
@@ -4,6 +4,7 @@ products:
     name: Casdon TD Pro 2
 primary_entity:
   entity: climate
+  translation_only_key: casdon_oven
```
Comment (make-all) [inline custom_components/tuya_local/devices/casdon_td_pro_2.yaml:7]: I think just "oven" is better, as other ovens are likely to have similar presets.  Even if some are different, it is no problem having extra presets defined in the translations.

Unlike the error codes, these do not have the "E1" prefix tying them strongly to a specific device implementation.

**Comment by make-all** [comment]
Maybe also add an icon for the "oven" climate translation key to icons.json.

mdi:microwave, mdi:toaster-oven and mdi:stove may be suitable candidates.

File: custom_components/tuya_local/devices/casdon_td_pro_2.yaml (hunk 2 above)
Comment (jianyun8023) [reply custom_components/tuya_local/devices/casdon_td_pro_2.yaml:7]: I think using an oven is also feasible; I’m just not sure if this list will become particularly large in the future.

**Pull request merged**